src/helpers/fhirUtils.js

```diff
@@ -56,11 +56,26 @@ const getBundleResourcesByType = (bundle, type, context = {}, first) => {
   return first ? null : [];
 };
 
+const getBundleEntriesByResourceType = (bundle, type, context = {}, first) => {
```

Comment:
DP1 : super small, but could we, for the sake of prettiness, make false the default value for first?

```suggestion
const getBundleEntriesByResourceType = (bundle, type, context = {}, first=false) => {
```

Reply:
I'm not really reviewing this PR so feel free to ignore me trespassing, but if we do that, can we make that change for `getBundleResourcesByType` as well?

Reply:
yes please.

Reply:
Sure! I think I was going off of the `getBundleResourcesByType` function, but I can update both!